src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        int randomNumber = new Random().nextInt(100);
+        ColorSupplier fff = new ColorSupplier();
+        Ball bigball = new Ball();
+        bigball.setColor(fff.toString());
```

Comment:
According to our checklist, use `name()` for getting `String` representation of enum constants. Don't use `toString()` or `String.valueOf()`. So, it's better to use `fff.get().name()` instead of `fff.toString()`.